modules/300-prometheus/images/grafana-dashboard-provisioner/hooks/dashboard_provisioner.sh

```diff
@@ -67,17 +56,16 @@ function __main__() {
       file="${title}.json"
     fi
 
-    mkdir -p "/tmp/dashboards/${folder}"
-    jq -rc '.definition' <<<${dashboard} >"/tmp/dashboards/${file}"
+    mkdir -p "${tmpDir}/${folder}"
+    jq -rc '.definition' <<<${dashboard} > "${tmpDir}/${file}"
   done
 
   if [[ "x${malformed_dashboards}" != "x" ]]; then
-    echo "Some dashboards are malformed: ${malformed_dashboards}"
-    exit 1
+    echo "Skipping malformed dashboards: ${malformed_dashboards}"
```

Comment:
I don't think that we have to drop dashboard generation on malformed one. Malformed dashboards are skipped and would not be generated